Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityController.cs

```diff
@@ -238,7 +238,7 @@ private void UpdateMenuData(InteractionSourceState interactionSourceState, Mixed
         /// Ensure that if a controller model was desired that we have attempted initialization.
         /// </summary>
         /// <param name="interactionSourceState">The InteractionSourceState retrieved from the platform.</param>
-        private void EnsureControllerModel(InteractionSourceState interactionSourceState)
+        public void EnsureControllerModel(InteractionSource interactionSource)
```

Comment:
Could this be made `internal` instead? It doesn't feel like something we want callable from anywhere.

```suggestion
        internal void EnsureControllerModel(InteractionSource interactionSource)
```